etc/rules-changelog.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
wanna just nuke this thing for now?
we can leave some of the code in, but make it dead or commented out?